src/pagination/pagination.ts

```diff
@@ -33,22 +33,32 @@ import {NgbPaginationConfig} from './pagination-config';
 
         <li *ngIf="directionLinks" class="page-item" [class.disabled]="!hasNext()">
           <a aria-label="Next" class="page-link" href (click)="!!selectPage(page+1)">
-            <span aria-hidden="true">&raquo;</span>
-            <span class="sr-only">Next</span>
+            <span aria-hidden="true">{{nextText}}</span>
+            <span class="sr-only">{{_nextTextSr}}</span>
           </a>
         </li>
         
         <li *ngIf="boundaryLinks" class="page-item" [class.disabled]="!hasNext()">
           <a aria-label="Last" class="page-link" href (click)="!!selectPage(pageCount)">
-            <span aria-hidden="true">&raquo;&raquo;</span>
-            <span class="sr-only">Last</span>
+            <span aria-hidden="true">{{lastText}}</span>
+            <span class="sr-only">{{_lastTextSr}}</span>
           </a>                
         </li>        
       </ul>
     </nav>
   `
 })
 export class NgbPagination implements OnChanges {
+  private _firstText;
```

Comment:
I think the non-SR (screen reader) properties need to be private because if they become public (by removing the underscore and making the properties public) we would get into an infinite loop in their corresponding setter methods. The template does not actually use the private property reference, it uses the public one: e.g. firstText.

It is true though that the template uses the private SR properties (e.g. _firstTextSr). I thought about making the SR properties public as well but then it felt to me like they would need the Input decorator and that they would need to be included in the public API and the docs. Adding 4 more properties to the UI felt too much to me at this point. Without the 4 additional public SR properties it is true that we won't have the flexibility of allowing the SR and non-SR values to be set to different values once the default configuration is overridden.

Maybe a good compromise for now is to change the description for the non-SR properties (and hence the API docs in the demo) to say something like this?

Text to display for the first page button. _This value will be used for both screen readers and non-screen reader displays. Default value: «« (for non-screen reader displays); First for screen-reader displays._

If users will end up needing in the future the ability to set the SR text to values different than the non-SR text we could expose the SR properties in the public properties when that need arises. 

On a slightly different note, I think I saw an API docs issue talking about the extraction of default values to include in the API docs. I hope that hard coding the default values in the property's comment won't interfere with the resolution of that issue.